src/tests/serverless/mockData.json

```diff
@@ -1,7 +1,9 @@
 {
-    "headers": {
-      "Content-Type": "application/json"
-    },
-    "body": "{\"name\": \"Frederic\"}",
-    "queryStringParameters": { "startDate": "2023-11-03", "endDate": "2021-11-03" }
-}
+  "headers": {
+    "Content-Type": "application/json"
+  },
+  "body": "{\"name\": \"Fredasdasderic\"}",
+  "pathParameters": {
+      "severaUserId": "83ebc1fe-16a4-b070-975f-e21c4334197b"
```

Comment:
Perhaps a more generic name e.g. TestUser. Also is this a real severUserId? If it is it should be removed and perhaps replaced with [insert severa id here].
Although this isn't a big deal, remember to be extra careful with committing any id's that refer to actual data in the future.